LDRH_choice.py

```diff
@@ -456,7 +456,10 @@ def run_staircase(task, operation=None):
         if operation: output_score = [output['score'] if i==order_dict[operation] else None for i in range(1,5)]
         else: output_score = output['score']
 
-        handler.addData(output['score'])
+        if use_posterior_matching: 
+            if operation: handler.addData(operation,output['score'])
```

Comment:
we need to pass either the operation name for math task and the task name for the other tasks. It's needed for the logistic model because the constants used are dependent on what task and operations are currently run. There's no such information currently being passed to the handler. The current handlers are separated by operation but it doesn't pass the operation information to the handler.